tasks/14.txt

```diff
@@ -1,3 +1,14 @@
 Create a SQL query to get Product list of stock is less than the quantity on order:
 
-| ProductName | UnitsInOrder| UnitsInStock |
\ No newline at end of file
+| ProductName | UnitsInOrder| UnitsInStock |
+
+db.products.aggregate([
+    { $project: {
+        ProductName: 1,
+        UnitsOnOrder: 1,
+        UnitsInStock: 1, 
+        check: {$gt: ["$UnitsOnOrder", "$UnitsInStock"]}
+    }},
+    { $match : { check: true }},
+    { $project: { check: 0, _id: 0 }}
+])
```

Comment:
Can be done with 1 match + 1 project. See
https://docs.mongodb.com/manual/reference/operator/query/expr/